### PR TITLE
Handle event home screen

### PR DESCRIPTION
**Pull request by chicky-dev**
https://edu-redmine.framgia.vn/issues/3558

### CODE REVIEW

**Comment by chicky-dev** [comment]
ready

**Comment by chicky-dev** [comment]
ready

File: app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/HomeViewModel.java [hunk 1]
```diff
@@ -2,29 +2,101 @@
...
+import io.reactivex.functions.Consumer;
+import io.reactivex.schedulers.Schedulers;
+
 public class HomeViewModel extends BaseObservable {
+    private static final String TAG = "HomeViewModel";
+    private static final int DEFAULT_PAGE = 1;
+    private int mPage;
+    private ObservableField<String> mGenreId;
     private HighLightAdapter mHighLightAdapter;
     private MovieAdapter mMovieAdapter;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/themoviedb/ui/home/HomeViewModel.java]: Không nên chứa Adapter chỉ nên chứa Data Object

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by chicky-dev** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**